### PR TITLE
Updated Sovrn Bid Adaptor for MultiSized and added Error Call Home.

### PR DESCRIPTION
**Pull request by jrosendahl**
## Type of change

- [ ] Bugfix
- [x] Feature
- [ ] New bidder adapter   
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [ ] Other

## Description of change

Updated Sovrn Adapter to support multisized ads and record errors

- contact email of the adapter’s maintainer
- [x] official adapter submission

For any changes that affect user-facing APIs or example code documented on http://prebid.org, please provide:

- A link to a PR on the docs repo at https://github.com/prebid/prebid.github.io/

### CODE REVIEW

File: modules/sovrnBidAdapter.js [hunk 1]
```diff
@@ -72,48 +85,83 @@ export const spec = {
...
+  constructor(e, data) {
+    utils.logError(e)
+    this.error = {}
+    this.error.payload = 'error'
+    this.error.ts = utils.timestamp()
+    this.error.message = e.message
+    this.error.data = data
+    this.error.source = 'sovrnBidAdapter'
+    this.error.stack = e.stack
+    this.error.prebidVersion = $$PREBID_GLOBAL$$.version
```
Comment (mike-chowla) [inline modules/sovrnBidAdapter.js]: Use of $$PREBID_GLOBAL$$ is not allowed.  Check the other other adapters as to how to get the version substituted in at build time.

File: modules/sovrnBidAdapter.js [hunk 2]
```diff
@@ -72,48 +85,83 @@ export const spec = {
...
+    this.error.url = utils.getTopWindowLocation().href
+    this.error.userAgent = navigator.userAgent
+  }
+  send() {
+    ajax(
```
Comment (mike-chowla) [inline modules/sovrnBidAdapter.js]: Adapters are not permitted to make their own network requests.  Generally what we suggest is sending the data via getUserSyncs

File: modules/sovrnBidAdapter.js (hunk 1 above)
Comment (jrosendahl) [reply modules/sovrnBidAdapter.js]: addressed in commit

File: modules/sovrnBidAdapter.js (hunk 2 above)
Comment (jrosendahl) [reply modules/sovrnBidAdapter.js]: addressed in commit

**Comment by jrosendahl** [comment]
I have addressed the commit data.  Mike if you are too busy to do this review, would it help if I split multisized support and error tracking into two PR's?

**Comment by mkendall07** [comment]
This pull request **introduces 1 alert** when merging f165cb0c11b2eb28c3c0e98ac51011e19d592ea9 into 2c5685cb271c364478113546cef471c11499ec06 - [view on LGTM.com](https://lgtm.com/projects/g/prebid/Prebid.js/rev/pr-4b400237ea00ea80c78e9dbd8ffae65b5fe263d7)

**new alerts:**

* 1 for Missing &#39;this&#39; qualifier

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by jrosendahl** [comment]
So, is there something I need to do to get this merged?  

Please let me know if there is anything I can do to help move this forward.

**Comment by jsnellbaker** [comment]
@jrosendahl Can you rebase off of master?  There were some updates made to the test directory and various test files that should address this error.

**Comment by jrosendahl** [comment]
Coverage is now at 91% for module.

**Pull request merged**